swan/scripts/before-notebook.d/02_environment.sh

```diff
@@ -130,6 +130,28 @@ if [ "$SOFTWARE_SOURCE" == "lcg" ]; then
   fi
 
   _log "Finished setting up CVMFS and user environment"
+else
+  # Configure NXCALS for customenvs
+  export SPARK_HOME=/cvmfs/sft.cern.ch/lcg/releases/spark/3.5.1-dfe72/x86_64-el9-gcc13-opt
+
+  # Configure necessary settings (it doesn't run without this)
```

Comment:
This comment does not give any information. Why is the code below actually needed?

Reply:
Actually, now that I read the code I don't get why it is needed. This is just loading extensions you previously loaded in other SWAN sessions, as per a file that got stored in your EOS user space. We should not need all this.